### PR TITLE
[xml] Update Romanian translation for Notepad++ 8.7.2

### PR DESCRIPTION
**Pull request by AndreiMiloiu**

### CODE REVIEW

**Comment by donho** [comment]
@AndreiMiloiu 
It's not normal to me that the localization has a lot of small ajustations during 1 year:
https://github.com/notepad-plus-plus/notepad-plus-plus/commits/master/PowerEditor/installer/nativeLang/romanian.xml

Could do your major modification in this PR, then there are only added or deleted items in the future PR?

**Comment by AndreiMiloiu** [comment]
These are the final adjustments. I hope there is no mistake, typo or untranslated string left.

**Review by donho — CHANGES_REQUESTED** [review]
New added items for translations are not present. Please check the updated english.xml

**Comment by AndreiMiloiu** [comment]
Done! I know I promised I don't have anything to change anymore. However, I found some small corrections to do and I did them. I hope everything is good now.

File: PowerEditor/installer/nativeLang/romanian.xml [hunk 1]
```diff
@@ -1741,14 +1742,14 @@ Găsește în toate fișierele, dar exclude toate dosarele log sau logs în mod
...
             <find-status-scope-forward value="de la semnul de omisiune la sfârșitul fișierului"/>
-            <finder-find-in-finder value="Căutare în rezultatele găsite..."/>
+            <finder-find-in-finder value="Caută în rezultatele găsite..."/>
             <finder-close-this value="Închide căutătorul"/>
-            <finder-collapse-all value="Restrânge toate"/>
-            <finder-uncollapse-all value="Extinde toate"/>
+            <finder-collapse-all value="Restrânge-le pe toate"/>
+            <finder-uncollapse-all value="Extinde-le pe toate"/>
             <finder-copy value="Copiere"/>
 			<finder-copy-paths value="Copiază numele căii(căilor) selectat(e)"/>
```
Comment (donho) [inline PowerEditor/installer/nativeLang/romanian.xml:1750]: Need to change

File: PowerEditor/installer/nativeLang/romanian.xml [hunk 2]
```diff
@@ -1741,14 +1742,14 @@ Găsește în toate fișierele, dar exclude toate dosarele log sau logs în mod
...
             <finder-clear-all value="Golește-le toate"/>
-            <finder-open-all value="Deschide-le toate"/>
+            <finder-open-all value="Deschide calea (căile) selectat(e)"/>
```
Comment (donho) [inline PowerEditor/installer/nativeLang/romanian.xml:1752]: Need to change

File: PowerEditor/installer/nativeLang/romanian.xml [hunk 3]
```diff
@@ -1327,7 +1328,7 @@
                 <MISC title="DIVERSE">
                     <ComboBox id="6347">
```
Comment (donho) [inline PowerEditor/installer/nativeLang/romanian.xml:1329]: Need to add one more item.

**Review by donho — CHANGES_REQUESTED** [review]
Please check the add item in english.xml on master to update your localization file!
Also make sure all the translation has been modified as you want for once: because all the unnecessary modified items won't be accepted in the future PR.

**Comment by donho** [comment]
@AndreiMiloiu 
Have all the modifications been done?

**Comment by AndreiMiloiu** [comment]
Yes! All of them!

**Review by donho — APPROVED** [review]

**Pull request closed without merging**